pkg/gpt/data.go

```diff
@@ -0,0 +1,266 @@
+// This file is a GPT of a ChromeOS file system, which has
```

Comment:
I don't like this but I can't think of a better way to do this. So I guess we're stuck with this file

Reply:
Yeah chromeos is a pretty good use of GPT, so why not.

In fact this header found a bug for me: the backup header has a higher address than the backup partition table. You tend to assume the ordering in the file is GPT header, then GPT partitions, as in the start. Not so. So it's been helpful to use a chromeos header. Real world is good.

Reply:
Haha I mean that we're basically checking in a binary blob in source form here. But like I said I don't see a way to do it better.